client/src/components/DataFiles/DataFilesProjectsList/DataFilesProjectsList.js

```diff
@@ -19,9 +23,12 @@ const DataFilesProjectsList = ({ modal }) => {
       ? 'PROJECTS_GET_LISTING'
       : 'PROJECTS_SHOW_SHARED_WORKSPACES';
     dispatch({
-      type: actionType
+      type: actionType,
+      payload: {
+        queryString: modal ? null : query.query_string
+      }
     });
-  }, [dispatch]);
+  }, [dispatch, query.query_string]);
```

Comment:
> When you clear any search input box by clicking the x, it doesn't take you back to the complete listing for the area. this is consistent across all searchable areas, but should it work this way?

This is consistent across all searches. We should fix it in a PR that also generalizes search bars so that we don't have to have 3 different implementations of a mostly-identical component.